azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -39,6 +39,9 @@
 
 public class ExecutableFlow extends ExecutableFlowBase {
 
+  public static final int DEFAULT_FLOW_RETRY_LIMIT = 2;
```

Comment:
Clarification: I was trying to see if we are recording these retry values in the event reporter. Are we recording this to be able to query and get numbers like "how many times we end up retrying due to system? how many times user specified retries?" etc